### DISCUSSION TITLE
Gigabyte RGBFUSION!

### DISCUSSION THREAD

**Discussion by svuddi** [ChromaControl/ChromaControl#342] [Q&A] [answered]
Any idea how to link GB RGBFUSION or steelseries GG to razer studio? 
As I've a GB mobo, asus GPU, Razer mouse/kb, steelseries headset and corsair ram, ever since i bought my pc and i had its lights off due to the crazy conflicts between lighting sync softwares, but I just landed here and i'm using your tool and tbh I'm surprised seeing corsair in my razer studio, asus gpu not working but i'm gonna try your method of fixing it, 

if i go back in time i'd just get an asrock mobo like the one i had before and sync everything out through razer studio, it was perfect 
![image](https://github.com/user-attachments/assets/ad9e177c-e26b-452b-a310-7d665345215c)

**Comment by andrewbabbittdev** [comment, marked as answer]
This is available in the latest development bits in our discord server. You will have to uninstall all gigabyte software before using it as it's powered by OpenRGB.

**Reply by svuddi** [reply]
duddddddeeee it's working fine and smooth as hellllllllll, I jumped straight into your discord and installed that new version, and it's even working without uninstalling GCC, I just made sure I exit it before installing, it's so clean bro, finally im controlling everything through oneplace, thanks isn't enough

**Reply by svuddi** [reply]
Hope that u make them splitted so if iwant to control ram only or gpu or other stuff, cuz now i control them all, but i'm happy anyways great work
![image](https://github.com/user-attachments/assets/b4a80ee1-04af-497a-b553-83ffb1a609ab)
